### ISSUE TITLE
Chat Panel wrong initial aria message

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#26069]
Testing -186218

When focused on the chat panel the first time, NVDA read `Chat input: type code and press enter to run`. I don't think it's correct to call it "type code", it should be mentioning chat or conversation.
